src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -1007,6 +1010,19 @@ query_path(const char *szInput, int *is_target_path, dfs_obj_t **parent, char *i
 		/* trying to avoid lock as much as possible */
 		if (!daos_inited) {
 			/* daos_init() is expensive to call. We call it only when necessary. */
+			int low_fd_list[DAOS_MIN_FD], low_fd_count = 0, fd_kernel, idx;
+
+			fd_kernel = open("/proc/self/maps", O_RDONLY);
```

Comment:
What's wrong with opening "/"? 

Presumably this code can only get here after the hooks have been installed so perhaps it would be better to call the redirected open directly?